### PR TITLE
Adds swagger documentation

### PR DESCRIPTION
**Pull request by Imamah-Zafar**
## Description

Add swagger documentation

## Testing Instructions

Run the api server and open URL `http://localhost:3000/docs` on the browser to view the swagger doc

## Additional Notes

[Add any additional notes or context]

### CODE REVIEW

File: backend/backend/backend-src/docs/api/events/get-event-by-address.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            schema: {
+              type: 'object',
+              properties: {
+                status: {type: 'string', example: 'ok'},
+                events: {
+                  type: 'array',
+                  items: {
+                    type: 'object',
+                    properties: {
+                      // Define properties for events
```
Comment (noumanahmads) [inline backend/backend/backend-src/docs/api/events/get-event-by-address.ts:29]: @Imamah-Zafar  
I think something is missing here.

File: backend/backend/backend-src/docs/api/events/get-event.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+          'application/json': {
+            schema: {
+              type: 'object',
+              properties: {
+                status: {type: 'string', example: 'ok'},
+                events: {
+                  type: 'array',
+                  items: {
+                    type: 'object',
+                    properties: {},
```
Comment (noumanahmads) [inline backend/backend/backend-src/docs/api/events/get-event.ts:37]: @Imamah-Zafar 
Please add properties here ?

File: backend/backend/backend-src/docs/api/package/cancel-package.ts [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                type: 'string',
+                example: 'https://example.com/package.zip',
+              },
+              packageName: {
+                type: 'string',
+                example: 'test-package',
+              },
+              packageRoles: {
+                type: 'object',
+                example: {},
```
Comment (noumanahmads) [inline backend/backend/backend-src/docs/api/package/cancel-package.ts:52]: @Imamah-Zafar 
This must have properties?

**Comment by fariedt** [comment]
I had separated out the address events (user address) and package events because they fetch data from different tables.  Does the new API know which table to query from?

**Review by noumanahmads — CHANGES_REQUESTED** [review]

File: backend/backend/backend-src/docs/api/events/get-event-by-address.ts (hunk 1 above)
Comment (fariedt) [reply backend/backend/backend-src/docs/api/events/get-event-by-address.ts:29]: This endpoint needs to distinguish between events stored against user addresses and events stored against package addresses.  @asimm241

**Review by fariedt — CHANGES_REQUESTED** [review]

File: backend/backend/backend-src/docs/api/events/get-event-by-address.ts (hunk 1 above)
Comment (Imamah-Zafar) [reply backend/backend/backend-src/docs/api/events/get-event-by-address.ts]: in the recent commit i have updated the URL so we have 
```/v2/events/package/:address/:ticker``` and ```'/v2/events/address/:address``` endpoints to fetch package event and address event respectively

**Review by fariedt — APPROVED** [review]

**Pull request merged**